usertable.cpp

```diff
@@ -327,6 +378,34 @@ void UserTable::OnEvent(InotifyEvent& rEvt)
   // discard event if user has no access rights to watch path
   if (!(m_fSysTable || MayAccess(pW->GetPath(), DONT_FOLLOW(rEvt.GetMask()))))
     return;
+    
+  //#if 0
+  // log output for each dir + file + event
+  std::string events;
+  rEvt.DumpTypes(events);
+  syslog(LOG_INFO, "PATH (%s) FILE (%s) EVENT (%s)", pW->GetPath().c_str() , IncronTabEntry::GetSafePath(rEvt.GetName()).c_str() , events.c_str());
+  //#endif
+  
+  // add new watch for newly created subdirs
+  if ( rEvt.IsType(IN_ISDIR) && (rEvt.IsType(IN_CREATE) || rEvt.IsType(IN_MOVED_TO)) )
+  {
+	Dispose();
+	sleep (1);
```

Comment:
I've said it before I think, but introducing that `sleep(1);` here, you between a `Dispose()` and `Load()` can potentially result in incron missing events while it is sleep. I find it to be a very concerning change.